internal/kafka/internal/presenters/cluster.go

```diff
@@ -0,0 +1,59 @@
+package presenters
+
+import (
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/api/public"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/internal/kafka/internal/services"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/api"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/client/observatorium"
+	"github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pkg/errors"
+)
+
+func PresentEnterpriseCluster(cluster api.Cluster, observabilityConfiguration *observatorium.ObservabilityConfiguration, kasFleetshardOperatorAddon services.KasFleetshardOperatorAddon) (public.EnterpriseCluster, *errors.ServiceError) {
+	params := []public.EnterpriseClusterParameter{
+		{
+			Id:    "gateway",
+			Value: observabilityConfiguration.RedHatSsoGatewayUrl,
+		},
+		{
+			Id:    "tenant",
+			Value: observabilityConfiguration.RedHatSsoTenant,
+		},
+		{
+			Id:    "redHatSsoAuthServerUrl",
+			Value: observabilityConfiguration.RedHatSsoAuthServerUrl,
+		},
+		{
+			Id:    "redHatSsoRealm",
+			Value: observabilityConfiguration.RedHatSsoRealm,
+		},
+		{
+			Id:    "metricsClientId",
+			Value: observabilityConfiguration.MetricsClientId,
+		},
+		{
+			Id:    "metricsSecret",
+			Value: observabilityConfiguration.MetricsSecret,
+		},
+	}
+
+	fsoParams, svcErr := kasFleetshardOperatorAddon.GetAddonParams(&cluster)
+
+	if svcErr != nil {
+		return public.EnterpriseCluster{}, svcErr
+	}
+
+	for _, param := range fsoParams {
+		params = append(params, public.EnterpriseClusterParameter{
+			Id:    param.Id,
+			Value: param.Value,
+		})
+	}
+
+	c := public.EnterpriseCluster{
+		ClusterId:  cluster.ClusterID,
+		Status:     cluster.Status.String(),
+		Parameters: params,
```

Comment:
Let me know what you think of https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1429#discussion_r1039495248

Reply:
sure. I can separate them. btw. one thing I noticed is inconsistency in the id names format, e.g.:

```

{
    "cluster_id": "****",
    "status": "cluster_accepted",
    "parameters": [
        {
            "id": "gateway",
            "value": "****"
        },
        {
            "id": "tenant",
            "value": "****"
        },
        {
            "id": "redHatSsoAuthServerUrl",
            "value": "****"
        },
        {
            "id": "redHatSsoRealm",
            "value": "****"
        },
        {
            "id": "metricsClientId",
            "value": "****"
        },
        {
            "id": "metricsSecret",
            "value": "****"
        },
        {
            "id": "sso-auth-server-url",
            "value": "****"
        },
        {
            "id": "sso-client-id",
            "value": "****"
        },
        {
            "id": "sso-secret",
            "value": "****"
        },
        {
            "id": "control-plane-url",
            "value": "****"
        },
        {
            "id": "cluster-id",
            "value": "****"
        },
        {
            "id": "poll-interval",
            "value": "****"
        },
        {
            "id": "resync-interval",
            "value": "****"
        }
    ]
}

```

Reply:
> one thing I noticed is inconsistency in the id names format,

let's keep the id names as they are since the parameters are for two separate Operators: Fleetshard and Observability. 
So they'll be two separate secrets where the id will be the `key` in the secret and each Operator expect them in different formats.